src/runtime.rs

```diff
@@ -137,11 +137,23 @@ where
         RevisionGuard::new(&self.shared_state)
     }
 
+    /// The unique identifier attached to this `SalsaRuntime`. Each
+    /// forked runtime has a distinct identifier.
     #[inline]
-    pub(crate) fn id(&self) -> RuntimeId {
+    pub fn id(&self) -> RuntimeId {
```

Comment:
I think we need to re-export `RuntimeId` in root as well.

Reply:
Yes....I thought I did that for some reason.